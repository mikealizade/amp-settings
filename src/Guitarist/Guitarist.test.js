import React, { Fragment } from 'react';
// import ReactDOM from 'react-dom';
// import App from './App';
// import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('<Guitarist>', () => {
  it('renders correctly', () => {
    const name = 'Joe Satriani';
    const song = 'Surfing With The Alien';

    const tree = renderer
      .create(
        <h1>{name && name} {name && <span>({song})</span>}</h1>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
