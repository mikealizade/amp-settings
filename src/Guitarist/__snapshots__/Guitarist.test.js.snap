// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Guitarist> renders correctly 1`] = `
<h1>
  Joe Satriani
   
  <span>
    (
    Surfing With The Alien
    )
  </span>
</h1>
`;
